src/main/java/org/apache/fineract/api/OperationsDetailedApi.java

```diff
@@ -227,74 +227,98 @@ public Page<TransactionRequest> transactionRequests(
      * @param page the count/number of page which we want to fetch
      * @param size the size of the single page defaults to [10000]
      * @param sortedOrder the order of sorting [ASC] or [DESC], defaults to [DESC]
-     * @param filterBy type of filter we want to apply, @see [Filter]
-     * @param ids the list of ids to match the query with
      */
     @PostMapping("/transactionRequests/export")
     public Map<String, String> filterTransactionRequests(
             HttpServletResponse response,
             @RequestParam(value = "page", required = false, defaultValue = "0") Integer page,
             @RequestParam(value = "size", required = false, defaultValue = "10000") Integer size,
             @RequestParam(value = "sortedOrder", required = false, defaultValue = "DESC") String sortedOrder,
-            @RequestParam(value = "filterBy", defaultValue = "TRANSACTIONID") String filterBy,
-            @RequestBody(required = true) List<String> ids) {
+            @RequestParam(value = "startFrom", required = false) String startFrom,
+            @RequestParam(value = "startTo", required = false) String startTo,
+            @RequestParam(value = "state", required = false) String state,
+            @RequestBody Map<String, List<String>> body) {
```

Comment:
@danishjamal104 : Can you communicate the updated API documentation to Chiranjiv?